### ISSUE TITLE
[1.10.2] [Suggestion] Could use better button placement

### ISSUE THREAD

**Issue by SnowShock35** [PrinceOfAmber/Cyclic#104]
http://ss.snowshock35.com/2016/09/1475021939-0d30722e.png

In regards to the above screenshot from the current version of the mod (1.10.2-1.7.7) I think the button location/accessibility should be re-thought as of right now it's squashed up at the top almost un-noticeable. Perhaps in a inventory tab? (Similar to how tinkers did it before) or just a more clear button placement)

**Comment by SnowShock35** [comment]
P.S. Sorry for the rough/ messy report. It's like 1am and I've been issue reporting since 11am.

**Comment by PrinceOfAmber** [comment]
Thanks for the report. When i tried to add tabs last time, it crashed when other mods added them so i was missing something.  My idea was to have it similar to the baubles button, so just in top left instead of top right. I should look into tabs again.

**Comment by PrinceOfAmber** [comment]
For now i will make them a bit bigger.

There is no standard built in way to add survival tabs the way there is for Creative tabs.  If i tried to make my buttons into real tabs, it would just conflict and overlap tabs from other mods, maybe even crash. 

I googled and searched forums.  If there was an API or pattern or right way to do this i would love to know.  Source: http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/modification-development/2120245-forge-creating-survival-inventory-tabs

**Comment by SnowShock35** [comment]
@PrinceOfAmber Could try see how Tinkers' Construct does it?

**Comment by SnowShock35** [comment]
And thank you again :)

**Comment by PrinceOfAmber** [comment]
Yeah once i have time ill look into how they do it, its hopefully possible. like i said above, if there is no API or standard way, i wouldnt want to cause problems.  Anyway, new release is up   https://minecraft.curseforge.com/projects/cyclic/files/2333207
